src/main/java/com/launchdarkly/client/EvaluationError.java

```diff
@@ -0,0 +1,10 @@
+package com.launchdarkly.client;
+
+/**
+ * An error indicating an abnormal result from evaluating a feature
+ */
+public class EvaluationError extends Exception {
```

Comment:
A new exception to help us distinguish precondition / well-formedness failures during evaluation from `null` evaluation results, which should result in the `offVariation` being served.